testscript/cmd.go

```diff
@@ -409,6 +411,27 @@ func (ts *TestScript) cmdSymlink(neg bool, args []string) {
 	ts.Check(os.Symlink(args[2], ts.MkAbs(args[0])))
 }
 
+// cmdUNIX2DOS converts files from UNIX line endings to DOS line endings.
+func (ts *TestScript) cmdUNIX2DOS(neg bool, args []string) {
+	if neg {
+		ts.Fatalf("unsupported: ! unix2dos")
+	}
+	if len(args) < 1 {
+		ts.Fatalf("usage: unix2dos paths...")
+	}
+	for _, arg := range args {
+		filename := ts.MkAbs(arg)
+		data, err := ioutil.ReadFile(filename)
+		if err != nil {
+			ts.Fatalf("%s: %v", filename, err)
```

Comment:
```suggestion
			ts.Check(err)
```
The error from ReadFile already has the filename in it.

Reply:
Done.